lib/validators.js

```diff
@@ -16,7 +16,9 @@ exports.clientId = Joi.string()
 exports.clientSecret = Joi.string()
   .length(config.get('unique.clientSecret') * 2) // hex = bytes*2
   .regex(exports.HEX_STRING)
-  .required();
```

Comment:
this needs to come back, but removed because the validation rules are complex, needs to be fixed